keras_nlp/layers/modeling/masked_lm_head_test.py

```diff
@@ -119,6 +146,32 @@ def test_one_train_step(self):
         loss = model.train_on_batch(x=(token_data, position_data), y=label_data)
         self.assertGreater(loss, 0)
 
+    @parameterized.named_parameters(
```

Comment:
I would kill this test. Compiling a real loss function can make for slower tests, and with the parameterized testing this could slow down our suite.